src/test/scala/intent/runner/TestSuiteRunnerTest.scala

```diff
@@ -152,7 +152,7 @@ class TestSuiteRunnerTest extends TestSuite with State[TestSuiteTestCase] with
 /**
  * Wraps a runner for a specific test suite
  */
-case class TestSuiteTestCase(suiteClassName: String = null)(given ec: ExecutionContext) extends TestSuiteRunnerTester with
+case class TestSuiteTestCase(suiteClassName: String | Null = null)(given ec: ExecutionContext) extends TestSuiteRunnerTester with
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.